doghouse/server/ciutil/ciutil.go

```diff
@@ -100,10 +100,10 @@ func UpdateTravisCIIPAddrs(cli *http.Client) error {
 // IsFromAppveyor returns true if given request is from Appveyor.
 // https://www.appveyor.com/docs/build-environment/#ip-addresses
 func IsFromAppveyor(r *http.Request) bool {
-	return appveyorIPAddrs[ipFromReq(r)]
+	return appveyorIPAddrs[IPFromReq(r)]
 }
 
-func ipFromReq(r *http.Request) string {
+func IPFromReq(r *http.Request) string {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported function IPFromReq should have comment or be unexported